### ISSUE TITLE
What is the equivalent in Magick.net (Resize and Crop)

### ISSUE THREAD

**Issue by cssriraman** [dlemstra/Magick.NET#51]
> Dim img As New ImageMagickObject.MagickImage
> Call img.Convert("-resize", "390x", "-crop", "390x590+0+0", sourceImg, sourceImg)

I am currently using imageMagick com object and the above code. What would be the equivalent in Magick.net

Thanks

**Comment by cssriraman** [comment]
Ok. thanks.

How to change the crop position? It lookslike it starts from center of the image.

**Comment by dlemstra** [comment]
You will need to specify the `geometry` argument.

**Comment by erickao08** [comment]
Hi, 
How can I change the quality of resize image? 
It is different with the image that I resized with the command line. 

Thanks!

**Comment by dlemstra** [comment]
@erickao08 Is your question related to this post or are you executing a whole different command?

**Comment by erickao08** [comment]
@dlemstra 
Yes, my question is related to this post. I can open another issue if you prefer that way. 

I was able to use Resize() in c# to resize my image but the quality is bad; however when I used "convert input.jpg -resize 135 output.jpg" in cmd the quality is great. I am wondering if there is any attribute that I can tweak to make the quality the same. If you want I can show you an example

Thanks!
